aws/logs_monitoring/parsing.py

```diff
@@ -220,6 +220,10 @@ def s3_handler(event, context, metadata):
         if DD_MULTILINE_LOG_REGEX_PATTERN and multiline_regex_start_pattern.match(data):
             split_data = multiline_regex.split(data)
         else:
+            logger.debug(
+                "DD_MULTILINE_LOG_REGEX_PATTERN %s did not match start of file, splitting by line",
```

Comment:
This will also log when `DD_MULTILINE_LOG_REGEX_PATTERN` is not defined - not sure if that was intended

Reply:
oh good that you caught that